modules/monitoring/manifests/checks.pp

```diff
@@ -39,12 +39,15 @@
   }
 
   icinga::check { "check_whitehall_overdue_from_${::hostname}":
-    check_command       => 'check_whitehall_overdue',
-    service_description => 'overdue publications in Whitehall',
-    use                 => 'govuk_urgent_priority',
-    host_name           => $::fqdn,
-    notes_url           => monitoring_docs_url(whitehall-scheduled-publishing),
-    action_url          => "https://${whitehall_hostname}${whitehall_overdue_url}",
+    check_command              => 'check_whitehall_overdue',
+    service_description        => 'overdue publications in Whitehall',
+    use                        => 'govuk_urgent_priority',
+    host_name                  => $::fqdn,
+    notes_url                  => monitoring_docs_url(whitehall-scheduled-publishing),
+    action_url                 => "https://${whitehall_hostname}${whitehall_overdue_url}",
+    event_handler              => 'publish_overdue_whitehall',
+    attempts_before_hard_state => 2,
```

Comment:
@tijmenb I've just added a line to make this more explicit. Here it will check for overdue publications twice (before I made it explicit, it would check four times) with an interval of 10 seconds, triggering a `SOFT` alert if there are overdue publications before changing to the `HARD` state if necessary, which is when someone will be called.

Reply:
👍 awesome thanks, clear now!